docs/source/implementations/labeling_fixed_time_horizon.rst

```diff
@@ -0,0 +1,68 @@
+.. _implementations-labeling_fixed_time_horizon:
+
+====================
+Fixed Horizon Method
+====================
+
+Fixed horizon labels is a classification labeling technique used in the following paper: `Dixon, M., Klabjan, D. and
+Bang, J., 2016. Classification-based Financial Markets Prediction using Deep Neural Networks.
+arXiv:1603.08604. <https://arxiv.org/abs/1603.08604>`_
+
+Fixed time horizon is a common method used in labeling financial data, usually applied on time bars. The forward rate of return relative
+to :math:`t_0` over time horizon :math:`h` is calculated as follows:
+
+.. math::
+    r_{t0,t1} = \frac{p_{t1}}{p_{t0}} - 1
+
+Where :math:`t_1 = t_0 + h` is the time bar index after a fixed horizon of :math:`h` ticks have passed, and :math:`p_{t0}, p_{t1}`
```

Comment:
Please add here that you're referencing to **Advances in Financial Machine Learning**. Also, a Tip part can be added to describe additional information available in the book.